firefox-ios/Client/Telemetry/GleanUsageReporting.swift

```diff
@@ -48,6 +49,7 @@ final class GleanUsageReporting: GleanUsageReportingApi {
         GleanMetrics.Pings.shared.usageReporting.submit()
     }
 
+    @MainActor
     private func setUsageConstantValues() {
         GleanMetrics.Usage.os.set("iOS")
         GleanMetrics.Usage.osVersion.set(UIDevice.current.systemVersion)
```

Comment:
This whole chain of changes is because we're using `UIDevice.current.systemVersion` here